src/partner/modules/ui/Event/EventAddUI.tsx

```diff
@@ -0,0 +1,101 @@
+import styled from "@emotion/styled";
```

Comment:
emotion has been a pain in the ass for some devs, we are migrating from emotion to `styled-components` :'( sorry

Reply:
Btw, rename file to `EventAdd`, drop the `UI`